lib/modules/manager/crossplane/schema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const XPKGSchema = z.object({
```

Comment:
I find this const hard to read. It would be easier to read if there's a `-` to separate the abbreviation and the schema word, like this: `XPKG-Schema`. I think our convention is to always use camelCase though, so maybe this is the best that we can do.

Reply:
not valid as identifier